golem-worker-service/src/api/api_definition.rs

```diff
@@ -291,6 +317,134 @@ impl RegisterApiDefinitionApi {
         };
         record.result(response)
     }
+
+    /// Export an API Definition to OpenAPI
+    #[oai(
+        path = "/:id/:version/export",
+        method = "get",
+        operation_id = "export_api_definition"
+    )]
+    pub async fn export(
+        &self,
+        id: Path<ApiDefinitionId>,
+        version: Path<ApiVersion>,
+    ) -> Result<Json<OpenApiSpec>, ApiEndpointError> {
+        let record = recorded_http_api_request!(
+            "export_api_definition",
+            api_definition_id = id.0.to_string(),
+            version = version.0.to_string()
+        );
+
+        let response = {
+            // Retrieve the API definition
+            let api_definition = self
+                .definition_service
+                .get(
+                    &id.0,
+                    &version.0,
+                    &DefaultNamespace::default(),
+                    &EmptyAuthCtx::default(),
+                )
+                .instrument(record.span.clone())
+                .await?;
+
+            // Handle case where the API definition is not found
+            let compiled_definition = api_definition.ok_or_else(|| {
+                ApiEndpointError::not_found(safe(format!(
+                    "No API definition found for id {} and version {}",
+                    id.0, version.0
+                )))
+            })?;
+
+            // Convert `CompiledHttpApiDefinition` to `HttpApiDefinition`
+
+            let http_api_definition =
+                convert_compiled_to_http(compiled_definition).map_err(|err| {
+                    ApiEndpointError::internal(safe(format!(
+                        "Error converting API definition: {:?}",
+                        err
+                    )))
+                })?;
+
+            // Generate OpenAPI Specification
+            let openapi_spec = generate_openapi(&http_api_definition, &version.0.to_string());
+
+            Ok(Json(openapi_spec))
+        };
+
+        record.result(response)
+    }
+
+    #[oai(path = "/swagger-ui/*path", method = "get")]
+    async fn swagger_ui(&self, path: Path<String>) -> PlainText<String> {
+        PlainText(format!("Swagger UI for path: {}", path.0))
+    }
+}
+
+/// Configures the routes for the OpenAPI service, including `/docs` for Swagger and `/specs` for YAML.
+fn build_api_routes(services: Arc<Services>) -> Route {
+    let api_service = Arc::new(OpenApiService::new(
+        (
+            WorkerApi {
+                component_service: services.component_service.clone(),
+                worker_service: services.worker_service.clone(),
+            },
+            RegisterApiDefinitionApi::new(services.definition_service.clone()),
+            ApiDeploymentApi::new(services.deployment_service.clone()),
+            SecuritySchemeApi::new(services.security_scheme_service.clone()),
+            HealthcheckApi,
+        ),
+        "API Service",
+        "1.0",
+    ));
+
+    Route::new()
```

Comment:
I can only see the relative mounting point here. What's the absolute mounting point?